content/broker/broker-transition/getting-started/_index.nb.md

```diff
@@ -10,14 +10,14 @@ Weight: 1
 ## Tjenesteeiere
 For å bruke Formidling overgangsløsningen i Altinn 2 for å opprette, laste opp og hente fil metadata, må en tjenesteeier fullføre følgende trinn.
 1. Ha en eksisterende Altinn 2 Formidlingstjeneste.
-2. Ha eller lag en tilsvarende Altinn 3 Formidlings-ressurs. Se hvordan du kommer i gang med Altinn 3 Formidling [her](../../getting-started/).
-3. Be om et overgangsoppsett fra Altinn 2 tjenesten til Altinn 3 ressursen. Bestem dato for når dette skal gå i live. (For å minimere risiko for foreldreløse data i Altinn 2 lagring)
+2. Ha eller lag en tilsvarende Altinn 3 Formidlingstjeneste. Se hvordan du kommer i gang med Altinn 3 Formidling [her](../../getting-started/).
```

Comment:
"Ha eller lag en tilsvarende Altinn 3 Formidlingstjeneste."

"Ha eller lag en tilsvarende tjeneste i Altinn 3."

For å skille egennavnet "Altinn 3 Formidlingstjeneste" fra konseptet "en tjeneste som bruker Altinn 3 Formidlingstjeneste"